src/seedu/addressbook/commands/Command.java

```diff
@@ -37,6 +37,15 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     }
 
     /**
+     * Checks if the command have mutated data
+     * @return true for command types that mutate data
+     */
+
+    public boolean isMutating() {
+        return false;
+    }
+
+    /**
      * Executes the command and returns the result.
      */
     public CommandResult execute(){
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. missing space after `execute()`